src/js/components/modals/ServiceFormModal.js

```diff
@@ -26,6 +26,74 @@ const METHODS_TO_BIND = [
   'onMarathonStoreServiceCreateSuccess'
 ];
 
+const serverResponseMappings = {
+  'error.path.missing': 'Specify a path',
+  'error.minLength': 'Field may not be blank',
+  'error.expected.jsnumber': 'A number is expected',
+  'error.expected.jsstring': 'A string is expected'
+};
+
+const responseAttributePathToFieldIdMap = {
```

Comment:
Hey, you've stolen that from Marathon UI! 
Oh, both Apache License, ok, ok.
;))